src/chat/chat_panel.rs

```diff
@@ -33,6 +33,16 @@ live_design! {
     ICON_STOP = dep("crate://self/resources/icons/stop.svg")
     ICON_JUMP_TO_BOTTOM = dep("crate://self/resources/icons/jump_to_bottom.svg")
 
+    RoundedButton = <MoxinButton> {
```

Comment:
nitpick: Technically, MoxinButton is also rounded, right? Idk if there is a better name for this.